test/test.js

```diff
@@ -402,7 +404,7 @@ describe('FireWatch Compound Case', function () {
         this.timeout(10000);
         var tested = false;
 
-        var watcher = FireWatch.start( root, function () {
+        var watcher = FireWatch.start( root + '/', function () {
```

Comment:
Ah, I just wanna test the case which the first argument ends with slash, or we will add a new case for that?